tests/projects/test_project.py

```diff
@@ -587,29 +464,6 @@ def test_get_artifact_uri():
     assert uri == "store://models/arti/y:prod"
 
 
-def test_export_to_zip():
```

Comment:
Why is this test removed? Is it because you're using a hub function? Can't you mock it?

Reply:
edited: added mock